src/datepicker/__tests__/datepicker-unreliable.e2e.js

```diff
@@ -161,7 +161,7 @@ describe('Datepicker', () => {
 
     // input mask
     let selectedValue = await page.$eval(selectors.input, input => input.value);
-    expect(selectedValue).toBe('    /  /  ');
+    expect(selectedValue).toBe(' –     /  /  ');
```

Comment:
I don't think there are any range datepickers in https://github.com/uber/baseweb/blob/9197fe57e2abb1276ca80dbd59344f6601088042/src/datepicker/__tests__/datepicker.scenario.js Why do you think we're getting a hyphen here?